stdlib/std.jsonnet

```diff
@@ -1703,6 +1703,12 @@ limitations under the License.
 
   sum(arr):: std.foldl(function(a, b) a + b, arr, 0),
 
+  avg(arr)::
+    if std.length(arr) == 0 then
+      0
```

Comment:
Don't you want an error on this case like the other ones?

Reply:
Resolved this. Thanks for pointing it out.